fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/EndpointServlet.java

```diff
@@ -30,13 +57,14 @@
      */
     @Inject
     public EndpointServlet(ObjectMappersSuite objectMappers) {
-        this.objectMappers = objectMappers;
+        this(objectMappers, () -> Response.status(Response.Status.OK));
```

Comment:
What if we want to return non-OK response afterwards? Should we defer the status assignment?

Actually I've seen line 146 executing the same OK assignment below

Reply:
This response builder should only be used on successes.  But we can look at that more later.